vxfreeswitch/voice.py

```diff
@@ -494,6 +495,7 @@ def dial_outbound(self, to_addr):
             call_uuid = reply.args[1]
         if self.config.wait_for_answer:
             self._unanswered_channels[call_uuid] = Deferred()
+        self._msisdn_mapping[call_uuid] = to_addr
```

Comment:
This isn't the vumi message id, this is the special session ID that freeswitch gives us. Multiple vumi message ids can belong to a single session ID (the session id stays the same for the entirety of the call).

So we store the mapping from this session/call uuid to the msisdn, so whenever freeswitch comes back to us with something from the user, we have a look at the session uuid that freeswitch gives us, and we can immediately see what the msisdn is related to this session.